cpp/cmake/deps/openslide.cmake

```diff
@@ -22,7 +22,9 @@ if (NOT TARGET deps::openslide)
         set(OPENSLIDE_LIB_PATH "$ENV{CONDA_PREFIX}/lib/libopenslide.so")
     elseif (EXISTS /usr/lib/x86_64-linux-gnu/libopenslide.so)
         set(OPENSLIDE_LIB_PATH /usr/lib/x86_64-linux-gnu/libopenslide.so)
-    else () # CentOS 6
+    elseif (EXISTS /usr/lib/aarch64-linux-gnu/libopenslide.so)
+        set(OPENSLIDE_LIB_PATH /usr/lib/aarch64-linux-gnu/libopenslide.so)
```

Comment:
question: I can see why this change was needed in general, but it doesn't seem relevant to this PR. Was it just snuck in out of convenience?

Reply:
It is needed here or the arm64 wheel builds fail to find libopenslide and fail during the `run build_local` call. The use of `libopenslide` is only needed for C++ lib benchmarks and test cases and it isn't directly used by the Python wheel, but it has to be found to run the C++ build stage (There is not currently a CMake flag to disable building the benchmark binary or tests).